document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AbridgedAccountsDataHandler.java

```diff
@@ -52,17 +52,18 @@
 
     /**
      * Get an Abridged accounts resource from the given resource link
-     * @param transaction the transaction data
      * @param resourceLink the resource link of the abridged accounts
      * @param requestId the id of the request
      * @return a populated {@link DocumentInfoResponse} object
      * @throws HandlerException throws a custom handler exception
      */
-    public DocumentInfoResponse getAbridgedAccountsData(Transaction transaction, String resourceLink, String requestId)
+    public DocumentInfoResponse getAbridgedAccountsData(String resourceLink, String requestId)
             throws HandlerException {
 
         Accounts accounts = getAccounts(resourceLink, requestId);
 
+        Transaction transaction = null;
```

Comment:
Is this going to generate a document without the company name and number or fail with a null transaction? If it's going to fail anyway, can you throw an UnsupportedOperationException until it's fixed (link it to the story ID)?

Reply:
abridged accounts is being ignored. So no one should generate a document, we don't have a story for this as it was stated this morning in the group meeting we were just going to get on with the work. I can add the exception if required.

Reply:
It's misleading to have an implementation that doesn't work with no explanation of why/what changes are needed.

Reply:
Yeah we are correcting abridged immediately after completing the small-full changes. We are just concentrating on the small-full changes to allow it to be tested and unblock the other team.

Reply:
Can you make the TODO in capitals please? It's a recognised task in the IDE:
<img width="729" alt="screen shot 2018-12-04 at 11 59 56" src="https://user-images.githubusercontent.com/13251956/49440701-3df24500-f7bc-11e8-8b64-ec7ad2b84181.png">